### PR TITLE
Merge Hotfix security fixes 211008 into 21.10.x

### PR DESCRIPTION
**Pull request by chgautier**
## Description

Merge hotfix-security-fixes-211008 branch into 21.10.x

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,26 +5,27 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 ** Variables.
 */
 def serie = '21.10'
-def stableBranch = "${serie}.x"
-def devBranch = "dev-${serie}.x"
+def stableBranch = "21.10.x"
+def devBranch = "dev-21.10.x"
```
Comment (Thebarda) [inline Jenkinsfile]: I think you can revert those changes because the `serie` variable is defined above

Comment (chgautier) [reply Jenkinsfile]: yup, I tried different things to make the pipeline run and those changes where serie is replaced can definitely be reverted

Comment (chgautier) [reply Jenkinsfile]: though there's no harm in leaving them as well, we have them hardcoded in the 22.04.x branch

Comment (Thebarda) [reply Jenkinsfile]: I think we can leave as it is. Just a question about best practises ^^

Comment (chgautier) [reply Jenkinsfile]: for sure, I wouldn't mind doing it either way :)

**Pull request merged**